labels and job updates to mention new labels (#39456)
kubernetes/website: content/en/docs/reference/labels-annotations-taints/_index.md

using finalizers.
{{< /note >}}

### job-name (deprecated) {#job-name}

Example: `job-name: "pi"`

Used on: Jobs and Pods controlled by Jobs

{{< note >}}
Starting from Kubernetes 1.27, this label is deprecated.

> Kubernetes 1.27 and newer will ignore this label and use the prefixed job-name label

Reviewer: ```suggestion
Kubernetes 1.27 and newer will ignore this label and use the prefixed job-name label.
```

{{< /note >}}

### controller-uid (deprecated) {#controller-uid}

Example: `controller-uid: "$UID"`

Used on: Jobs and Pods controlled by Jobs

{{< note >}}
Starting from Kubernetes 1.27, this label is deprecated.

> Kubernetes 1.27 and newer will ignore this label and use the prefixed controller-uid label

Reviewer: ```suggestion
Kubernetes 1.27 and newer will ignore this label and use the prefixed controller-uid label.
```

{{< /note >}}

### batch.kubernetes.io/job-name {#batchkubernetesio-job-name}

Example: `batch.kubernetes.io/job-name: "pi"`

Used on: Jobs and Pods controlled by Jobs

### batch.kubernetes.io/controller-uid {##batchkubernetesio-controller-uid}

Example: `batch.kubernetes.io/controller-uid: "$UID"`

Used on: Jobs and Pods controlled by Jobs

> 

Reviewer 2: Can we add some descriptions to the above two labels?

Reviewer 2: By the way, are they labels or annotations?

Author: Labels. And yep I’ll add some descriptions

Author: Added.

using finalizers.
{{< /note >}}

### job-name (deprecated) {#job-name}

Example: `job-name: "pi"`

Used on: Jobs and Pods controlled by Jobs

{{< note >}}
Starting from Kubernetes 1.27, this label is deprecated.
Kubernetes 1.27 and newer will ignore this label and use the prefixed job-name label.
{{< /note >}}

### controller-uid (deprecated) {#controller-uid}

Example: `controller-uid: "$UID"`

Used on: Jobs and Pods controlled by Jobs

{{< note >}}
Starting from Kubernetes 1.27, this label is deprecated.
Kubernetes 1.27 and newer will ignore this label and use the prefixed controller-uid label.
{{< /note >}}

### batch.kubernetes.io/job-name {#batchkubernetesio-job-name}

Example: `batch.kubernetes.io/job-name: "pi"`

Used on: Jobs and Pods controlled by Jobs

> This label is used as a user-friendly way to get pods corresponding to jobs. The job-name comes from the name of the job and allows for an easy way to get pods corresponding to a job.

Reviewer 2: ```suggestion
This label is used as a user-friendly way to get Pods corresponding to Jobs.
The job-name comes from the name of the Job and allows for an easy way to get Pods corresponding to the Job.
```

### batch.kubernetes.io/controller-uid {#batchkubernetesio-controller-uid}

Example: `batch.kubernetes.io/controller-uid: "$UID"`

Used on: Jobs and Pods controlled by Jobs

> This label is used as a programatic way to get all pods corresponding to the job.  

Reviewer 2: ```suggestion
This label is used as a programatic way to get all Pods corresponding to the Job.  
```

> The controller-uid is a unique identifer that gets set in the `Selector` field so the job controller can get all the corresponding pods.

Reviewer 2: ```suggestion
The controller-uid is a unique identifer that gets set in the `selector` field so the Job controller
can get all the corresponding Pods.
```

using finalizers.
{{< /note >}}

### job-name (deprecated) {#job-name}

Example: `job-name: "pi"`

Used on: Jobs and Pods controlled by Jobs

{{< note >}}
Starting from Kubernetes 1.27, this label is deprecated.

> Kubernetes 1.27 and newer ignore this label and use the prefixed job-name label.

Reviewer 2: ```suggestion
Kubernetes 1.27 and newer ignore this label and use the prefixed `job-name` label.
```

> Kubernetes 1.27 and newer will ignore this label and use the prefixed job-name label.

Reviewer 3: ```suggestion
Kubernetes 1.27 and newer ignores this label and use the prefixed job-name label.
```

{{< /note >}}

### controller-uid (deprecated) {#controller-uid}

Example: `controller-uid: "$UID"`

Used on: Jobs and Pods controlled by Jobs

{{< note >}}
Starting from Kubernetes 1.27, this label is deprecated.

> Kubernetes 1.27 and newer ignore this label and use the prefixed controller-uid label.

Reviewer 2: ```suggestion
Kubernetes 1.27 and newer ignore this label and use the prefixed `controller-uid` label.
```

{{< /note >}}

### batch.kubernetes.io/job-name {#batchkubernetesio-job-name}

Example: `batch.kubernetes.io/job-name: "pi"`

Used on: Jobs and Pods controlled by Jobs

> This label is used as a user-friendly way to get Pods corresponding to Jobs.

Reviewer 2: ```suggestion
This label is used as a user-friendly way to get Pods corresponding to a Job.
```

> The job-name comes from the name of the Job and allows for an easy way to get Pods corresponding to the Job.

Reviewer 2: ```suggestion
The `job-name` comes from the `name` of the Job and allows for an easy way to get Pods corresponding to the Job.
```

### batch.kubernetes.io/controller-uid {#batchkubernetesio-controller-uid}

Example: `batch.kubernetes.io/controller-uid: "$UID"`

Used on: Jobs and Pods controlled by Jobs

> This label is used as a programatic way to get all Pods corresponding to the Job.  

Reviewer 2: ```suggestion
This label is used as a programatic way to get all Pods corresponding to a Job.  
```

> This label is used as a programatic way to get all Pods corresponding to a Job.  

Reviewer 4: spelling nit.
```suggestion
This label is used as a programmatic way to get all Pods corresponding to a Job.  
```

Author: TY. Applied.

> The controller-uid is a unique identifer that gets set in the `selector` field so the Job controller

Reviewer 2: ```suggestion
The `controller-uid` is a unique identifer that gets set in the `selector` field so the Job controller
```
